config/config_test.go

```diff
@@ -474,13 +476,6 @@ func TestGetSubnetConfigsFromFlags(t *testing.T) {
 			},
 			expectedErr: nil,
 		},
-		"subnet is not tracked": {
-			givenJSON: `{"Gmt4fuNsGJAd2PX86LBvycGaBpgCYKbuULdCLZs3SEs1Jx1LU":{"validatorOnly":true}}`,
-			testF: func(require *require.Assertions, given map[ids.ID]subnets.Config) {
-				require.Empty(given)
-			},
-			expectedErr: nil,
-		},
```

Comment:
Should we keep this test as it's still relevant and should produce the same result as the new case `default config used when no config provided` ?